treeherder/intermittents_commenter/constants.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+TREEHERDER_USER_AGENT = 'treeherder/{}'.format(settings.SITE_HOSTNAME)
```

Comment:
Could you inline this where it is used? (we'll probably also want to consolidate the session/fetching logic with our existing helpers at some point, but that can wait until another PR). eg
`'User-Agent': 'treeherder/{}'.format(settings.SITE_HOSTNAME),`